### PR TITLE
Exposure Page

### PR DESCRIPTION
**Pull request by bliitzkrieg**
## Overview
Add an exposure page under Applications that exposes NFORS mobile application information and a way for users to get their departments IAC.

## GitHub Issues
- #442 

## Changes
- New Controllers
- New Views
- New endpoint

## Screenshots / Videos
Screen Shot 2020-04-08 at 10 11 09 PM

Screen Shot 2020-04-08 at 10 11 14 PM

## Note
- PR in NFORS repo must be merged first

## Steps to Test
- Ensure environment variables are NFORS environment variables are set
- Login
- Click Exposures under the Applications tab in side menu

### CODE REVIEW

File: server/api/exposure/exposure.controller.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import axios from 'axios';
+
+export async function exposure(req, res) {
+  try {    
+    const department = req.fireDepartment;
+    const id = department._id;
```
Comment (bliitzkrieg) [inline server/api/exposure/exposure.controller.js:6]: Does this _id match the department ID in NFORS. I only have test data so I'm not sure.

File: server/auth/auth.service.js [hunk 2]
```diff
@@ -148,6 +149,21 @@ export function belongsToFireDepartment(req, res, next) {
...
+    });
+
+    req.NFORS_TOKEN = data.token;
+  } catch(err) {
+    throw new ForbiddenError('Error authenticated NFORS');
+  } finally {
+    next();
+  }
+}
+
```
Comment (bliitzkrieg) [inline server/auth/auth.service.js:166]: I moved the NFORS authorization to a middleware that sets a variable on `req`

File: client/app/exposure/exposure-home/exposure-home.controller.js [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+  constructor($http) {
+    this.$http = $http;
+    this.errors;
+    this.accessCode;
+    this.fetchIAC();
+  }
+
+  fetchIAC() {
+    this.$http.get('/api/exposure/iac')
```
Comment (chopchop505) [inline client/app/exposure/exposure-home/exposure-home.controller.js:18]: Can you make this an angular resource?  See examples in client/compenents/api

File: server/api/exposure/exposure.controller.js [hunk 4]
```diff
@@ -0,0 +1,23 @@
+import axios from 'axios';
+
+export async function exposure(req, res) {
+  try {    
+    const department = req.fireDepartment;
+    const id = department.fd_id;
```
Comment (chopchop505) [inline server/api/exposure/exposure.controller.js]: You should query off of firecares_id

**Comment by bliitzkrieg** [comment]
This is ready for review/merge

**Pull request merged**